### PR TITLE
refactor(all): rename master to main

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Update references to master to be main.

## QA

n/a

## Fixes

Fixes: #3952.

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -18,15 +18,15 @@ jobs:
       - name: Initialise MAAS
         run: sudo maas init region+rack --maas-url=${{env.MAAS_URL}}/MAAS --database-uri maas-test-db:///
       - name: Install Cypress
-        uses: cypress-io/github-action@master
+        uses: cypress-io/github-action@v2
```
Comment (huwshimi) [inline .github/workflows/cypress.yml:21]: This wasn't strictly necessary it just seemed to be what the cypress docs suggested, but can revert this change if there's a reason for it to be on master.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ Related Launchpad maas issue in the form `lp#number`.
 
 ## Backports
 
-In general, please propose fixes against *master* rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.
+In general, please propose fixes against _main_ rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.
```
Comment (huwshimi) [inline .github/pull_request_template.md:37]: Not quite sure why prettier changed `*` to `_`.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -34,6 +34,10 @@ Related Launchpad maas issue in the form `lp#number`.
...
 
-Only bug and security fixes should be backported, new features should only land in master.
+Only bug and security fixes should be backported, new features should only land in main.
+
+## Screenshots
```
Comment (huwshimi) [inline .github/pull_request_template.md:41]: Just a driveby I've been meaning to add for ages.

Comment (petermakowski) [reply .github/pull_request_template.md:41]: Nice. 👍
